content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -20,6 +20,15 @@ deprecated API versions to newer and more stable API versions.
 
 ## Removed APIs by release
 
+### v1.27
+
+The **v1.27** release will stop serving the following deprecated API versions:
+
+#### HorizontalPodAutoscaler {#horizontalpodautoscaler-v127}
+
+The **autoscaling/v2beta2** API version of HorizontalPodAutoscaler will no longer be served in v1.27.
+
+* Migrate manifests and API clients to use the **autoscaling/v2** API version, available since v1.23.
```

Comment:
Are there any API changes? (I couldn't get confirmation on this).

Reply:
@josephburnett @gjtempleton , can you confirm if there are any API changes and if all existing persisted objects are accessible via the new API
I can add the following if they are true:

> * All existing persisted objects are accessible via the new API
> * No notable changes

Reply:
that is correct